docs/mkdocs.yml

```diff
@@ -17,8 +17,10 @@ theme:
   custom_dir: overrides
 nav:
   - Overview: 'index.md'
+  - Data Types: 'data-types.md'
```

Comment:
Should data types go after annotation + querying? It seems like it would flow better.